ddtrace/internal/runtime/runtime_metrics.py

```diff
@@ -140,7 +140,7 @@ def flush(self):
         with self._dogstatsd_client:
             for key, value in self._runtime_metrics:
                 log.debug("Writing metric %s:%s", key, value)
-                self._dogstatsd_client.distribution(key, value)
+                self._dogstatsd_client.gauge(key, value)
```

Comment:
IIUC a gauge will only show you the last collected value. My gut feeling is that some metrics (e.g. errors) should be treated as `count`s. Is this not done by other runtimes?